utils/base_test.py

```diff
@@ -3,10 +3,11 @@
 from selenium import webdriver
 from webdriver_manager.chrome import ChromeDriverManager
 from selenium.webdriver.chrome.service import Service
+from utils.screenshot_utils import save_screenshot
 
 
 class BaseTest(unittest.TestCase):
-    def _setUp(self, url=None) -> None:
```

Comment:
Why you don't like '->'?
:)

Reply:
I think in this case it's obvious and it looks cleaner lol